styling-and-themes/swatch-distribution.md

```diff
@@ -0,0 +1,101 @@
+| Turquoise | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-default/swatches/default-turquoise.css
+
+
+#### Bootstrap
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+Swatch | CDN |
+| ----------- | ----------- |
+| Main | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-bootstrap/swatches/bootstrap-main.css
+| Main Dark | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-bootstrap/swatches/bootstrap-main-dark.css
+| Nordic | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-bootstrap/swatches/bootstrap-nordic.css
+| Urban | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-bootstrap/swatches/bootstrap-urban.css
+| Vintage | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-bootstrap/swatches/bootstrap-vintage.css
+
+#### Material
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+Swatch | CDN |
+| ----------- | ----------- |
+| Main | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-material/swatches/material-main.css
+| Arctic | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-material/swatches/material-arctic.css
+| Nova | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-material/swatches/material-nova.css
+| Lime Dark | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-material/swatches/material-lime-dark.css
+| Main Dark | https://blazor.cdn.telerik.com/blazor/{{site.uiForBlazorLatestVersion}}/kendo-theme-material/swatches/material-main-dark.css
+
+
+### ZIP archive
+
+One of the ways to install Telerik UI for Blazor is to use the ZIP Archive. Its `styles` folder contains all of the above listed swtaches. You can download it from your [Telerik account](https://www.telerik.com/account/). [Read more for the ZIP Archive]({%slug installation/zip%})...
+
+### VS and VS Code Wizard
+
+You can use the Telerik Blazor Extensions to easily create a new project pre-configured to use the Telerik UI for Blazor components or convert your existing one to add them. Telerik Blazor Extensions are available for Visual Studio and Visual Studio Code.
+
+The **Create New Project** wizard for [Visual Studio]({%slug getting-started-vs-integration-new-project%}) and [Visual Studio Code]({%slug getting-started-vs-code-integration-new-project%}) allows you to also select the desired theme/swatch for your application. It provides the above list of the most commonly used swatches you can choose from.
+
+Once you've created the project, the selected theme will be distributed as static asset. You can find it in the `wwwroot/lib/blazor-ui/styles` folder of the application.
```

Comment:
Static asset means something from the NuGet package, no? Here we should be talking about a "local file" or something like that. Also, didn't we intend to include the swatches in the NuGet package? Sync with Yoana if necessary. Otherwise, we should highlight here that upgrading UI for Blazor requires manually replacing the CSS file.

Reply:
This indeed has to be "local file". In this case, when the theme is referred as local file, when upgrading the user will have to manually update it.
The NuGet package will not include the swatches, the ZIP will.